sdk/cpp/core/src/path/netconf_session.cpp

```diff
@@ -412,8 +411,9 @@ static string get_filter_payload(path::Rpc & ydk_rpc)
 {
     auto entity = ydk_rpc.get_input_node().find("filter");
     if(entity.empty()){
-        YLOG_ERROR("Failed to get entity node.");
-        throw(YInvalidArgumentError{"Failed to get entity node"});
+    	return string{};
+//        YLOG_ERROR("Failed to get entity node.");
```

Comment:
It may not be safe to comment out this exception. If the entity is empty, is it may be better to raise an exception instead of silently returning empty string?